mne/decoding/transformer.py

```diff
@@ -17,7 +18,10 @@
 
 
 class Scaler(TransformerMixin):
-    """Standardize data across channels.
+    u"""Standardize data across channels.
+
+    By default, this makes each time point (within each epoch) have
```

Comment:
Right, this makes it so that each time point, within each epoch, have mu=0 std=1 *for each channel type*. Is that the intent? Then all I need to do is update the docstring to say "within each channel type" or so.

It's a bit weird, though -- it seems like you'd want to operate across all time points, or across all epochs, or so... I would have naively expected at least the scaling portion to just compute the std across all times and epochs for each channel type, and scale all the data for each channel type by that value.